src/components/DeviceCheckAddress.js

```diff
@@ -27,35 +27,34 @@ class CheckAddress extends PureComponent<Props, State> {
     this.verifyAddress({ device, account })
   }
 
-  componentDidUnmount() {
-    if (this.sub) this.sub.unsubscribe()
```

Comment:
I think we might want either to keep that or to just have a flag to know if it's unmounted and not do a setState if it's gone

Reply:
plus one

Reply:
done